src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
```

Comment:
Avoid using spaces in `href` attributes. If the `href` is supposed to link to an ID on the page, it should start with a `#` followed by the ID. If it's a placeholder, consider using `#` or `javascript:void(0);`.